packages/cypher-builder/src/expressions/list/ListIndex.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CypherEnvironment } from "../../Environment";
+import type { Variable } from "../../references/Variable";
+import type { CypherCompilable } from "../../types";
+import type { ListExpr } from "./ListExpr";
+
+/** Access individual elements in the list
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/lists/)
+ * @group Expressions
```

Comment:
This class is not exported, so it doesn't really makes much sense to document it tbh :sweat_smile: